department-of-veterans-affairs/va.gov-cms
# VACMS-1088 proofing page content display

## Description
Enhances content proofing experience for Benefits Detail Pages
- Structured display of paragraphs
- Help text for content fields (not paragraph fields)
- Improved layout


## QA steps

1. Go to http://pr2140.ci.cms.va.gov/test-page-do-not-publish
 - [x] Check the overall page for readability, scanability.
 -  Check the individual paragraphs for
    - [x] Completeness - all paragraph fields are represented
    - [x] Clarity - field labels make sense, paragraph display maps to front end
1. Edit the page. Make various changes and save.
1. Go to the revisions tab and diff the revision you just created with the previous one.
   - [x] Make sure that all the changes you made are displayed in the diff.
1. Check other random benefits detail pages for clarity and readability.
1. Go to proofing screens of other content types that use paragraphs and confirm there are no significant changes in appearance
   - [x] Detail page
   - [x] Documentation page
   - [x] Health services listing page
   - [x] Benefits hub landing page
1. Go to http://pr2140.ci.cms.va.gov/test-page-do-not-publish and add a broken link in and FAQ.   Save the node 
   - [x] validate that the broken link has been scanned and flagged the node link report.
